yarn-project/simulator/src/avm/opcodes/arithmetic.ts

```diff
@@ -32,7 +31,7 @@ export abstract class ThreeOperandArithmeticInstruction extends ThreeOperandInst
 
 export class Add extends ThreeOperandArithmeticInstruction {
   static readonly type: string = 'ADD';
-  static readonly opcode = Opcode.ADD;
+  static readonly opcode = Opcode.ADD_8; // FIXME: needed for gas.
 
```

Comment:
What's this fixme? Is it noting that we need to set `opcode = Opcode.ADD_8` because of gas?

Reply:
because we should be able to omit this and just say "as u16" or whatever later?